### PR TITLE
Add communication parameters in the URDF

### PR DESCRIPTION
**Pull request by buschbapti**
This PR adds the robot ip address and ports in the URDF for communication with both the real robot and the simulator. This is the only way to pass those parameters to the hardware interface.

### CODE REVIEW

File: robots/panda_arm.urdf.xacro [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <?xml version='1.0' ?>
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="panda">
     <xacro:arg name="prefix" default="panda_"/>
+    <xacro:arg name="robot_ip" default="127.0.0.1"/>
```
Comment (domire8) [inline robots/panda_arm.urdf.xacro:4]: Is 127.0.0.1 and localhost and 0.0.0.0 all the same?

**Review by domire8 — APPROVED** [review]

File: robots/panda_arm.urdf.xacro (hunk 1 above)
Comment (buschbapti) [reply robots/panda_arm.urdf.xacro:4]: this was tested and works. Not sure if `0.0.0.0` is equivalent

**Comment by buschbapti** [comment]
I have corrected the quote typo here as this actually fits in that PR. Let me know when I can merge that

**Review by eeberhard — APPROVED** [review]
Thanks for fixing that quote bug. The rest was proven to work quite well!

**Review by domire8 — APPROVED** [review]
Good for me too

**Pull request merged**